test/e2e/framework/metrics_util.go

```diff
@@ -371,9 +373,13 @@ func HighLatencyRequests(c clientset.Interface, nodeCount int) (int, *APIRespons
 	for i := range metrics.APICalls {
 		latency := metrics.APICalls[i].Latency.Perc99
 		isListCall := (metrics.APICalls[i].Verb == "LIST")
+		isClusterScopedCall := (metrics.APICalls[i].Scope == "cluster")
 		isBad := false
 		if latency > apiCallLatencyThreshold {
-			if !isListCall || !isBigCluster || (latency > apiListCallLatencyThreshold) {
+			if !isListCall ||
+				!isBigCluster ||
+				(!isClusterScopedCall && latency > apiListCallLatencyThreshold) ||
+				(latency > apiClusterScopeListCallThreshold) {
```

Comment:
This is a really confusing logic statement. I suggest one if/switch to set the threshold, and then this if should read `if latency > threshold`.

Reply:
Changed it. LG?